packages/portalnetwork/src/wire/utp/PortalNetworkUtp/PortalNetworkUTP.ts

```diff
@@ -0,0 +1,495 @@
+import { Discv5, toHex } from '@chainsafe/discv5'
+import { toHexString } from '@chainsafe/ssz'
+import { Debugger } from 'debug'
+import { bufferToPacket, Packet, PacketType, randUint16, UtpSocket } from '..'
+import { SubNetworkIds } from '../..'
+import { HistoryNetworkContentKeyUnionType, PortalNetwork } from '../../..'
+import { sendFinPacket, sendSynAckPacket } from '../Packets/PacketSenders'
+import { BasicUtp } from '../Protocol/BasicUtp'
+import { HistoryNetworkContentRequest } from './HistoryNetworkContentRequest'
+
+type UtpSocketKey = string
+
+export enum RequestCode {
+  FOUNDNDCONTENT_WRITE = 0,
+  FINDCONTENT_READ = 1,
+  OFFER_WRITE = 2,
+  ACCECPT_READ = 3,
```

Comment:
```suggestion
  ACCEPT_READ = 3,
```
Also fix the places this enum is used.